decks/raptor_rogue.json

```diff
@@ -2,35 +2,33 @@
   "cards": [
     "spell_backstab",
     "spell_backstab",
-    "spell_cold_blood",
-    "spell_cold_blood",
-    "minion_abusive_sergeant",
-    "minion_abusive_sergeant",
-    "minion_leper_gnome",
-    "minion_leper_gnome",
-    "spell_eviscerate",
-    "spell_eviscerate",
+    "spell_evsicerate",
```

Comment:
This should be spell_eviscerate, otherwise, when you test the deck, it will crash.